lib/publish.js

```diff
@@ -76,6 +76,7 @@ async function publishChartToRegistry(configPath, registry, name, version) {
                 );
 
                 const chartArchive = stdout.split(":")[1].trim();
+                registry = registry.includes("oci://") ? registry : "oci://".concat(registry)
```

Comment:
`registry.includes("oci://")` will never be `true`, to be honest I liked stripping it off on the login operation because some people might specify it based on the updated helm docs

prefer not to use concat for appending string

```suggestion
                registry = registry.includes("oci://") ? registry : "oci://" + registry
```

or 

```suggestion
                registry = registry.includes("oci://") ? registry : `oci://${registry}`
```

Reply:
done